README.md

```diff
@@ -168,6 +170,43 @@ python3 ecosify-strings.py Extensions
 python3 ecosify-strings.py Shared
 ```
 
-## Release 🚀
+## :rocket: Release
 
 Follow the instructions from our [confluence page](https://ecosia.atlassian.net/wiki/spaces/MOB/pages/2460680288/How+to+release)
+
+## Howto update the release notes (semi-automated)
+
+Make sure that `fastlane` and `transifex`-cli is installed.
+
+### Add source release notes to transifex (English)
+
+- Make sure that an _inflight_ version exists in AppStore Connect. If not, create one.
+- Add English text to release notes in AppStore Connect
+- Download metadata from AppStore
+
+    ```bash
+    bundle exec fastlane deliver download_metadata
+    ```
+
+- Push source to Transifex (It is important to limit the push to release_notes.txt)
+
+    ```bash
+    tx push ecosia-ios-search-app.release_notestxt
```

Comment:
where does this file leave? as not mentioned, would it work if this command is run from the root project folder?